v2.0/alter-sequence.md

```diff
@@ -34,11 +35,50 @@ table td:first-child {
 
  Parameter | Description
 -----------|------------
+`IF EXISTS` | Modify the sequence only if it exists; if it does not exist, do not return an error.
+`sequence_name` | The name of the sequence you want to modify.
+`INCREMENT` | The new value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.
+`MINVALUE` | The new minimum value of the sequence.
+`MAXVALUE` | The new maximum value of the sequence.
+`START` | The value the sequence starts at if you `RESTART` or if the sequence hits the `MAXVALUE` and `CYCLE` is set. <br><br>`RESTART` and `CYCLE` are not implemented yet.
+`CYCLE` | The sequence will wrap around when the sequence value hits the maximum or minimum value. If `NO CYCLE` is set, the sequence will not wrap.
+`newname` | The new name of the sequence, which must be unique to its database and follow these [identifier rules](keywords-and-identifiers.html#identifiers). <br><br>Name changes do not propagate to the  table(s) using the sequence.
 
 ## Examples
 
-### Example 1
+### Change the Increment of a Sequence
+
+In this example, we're going to change the increment of a sequence from its current state (i.e., `1`) to `2`.
```

Comment:
Should it be "increment value"? "Change the increment" sounds odd.

Reply:
edited